osbenchmark/workload/params.py

```diff
@@ -945,6 +951,17 @@ def _update_request_params(self):
             self.PARAMS_NAME_ALLOW_PARTIAL_RESULTS, "false")
         self.query_params.update({self.PARAMS_NAME_REQUEST_PARAMS: request_params})
 
+    def _update_body_params(self, vector):
```

Comment:
Should query be updated as part of this PR?

Reply:
I was waiting for previous PR to merge. I rebased it now